baremaps-tile-postgres/src/test/java/com/baremaps/tile/postgres/PostgisQueryParserTest.java

```diff
@@ -16,74 +16,66 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
-import com.baremaps.config.tileset.Query;
+import com.baremaps.model.Query;
 import java.util.Optional;
 import org.junit.jupiter.api.Test;
 
 class PostgisQueryParserTest {
 
   @Test
   void parse1() {
-    parse(
-        new Query(0, 1, "SELECT id, tags, geom FROM table"),
-        "id",
-        "tags",
-        "geom",
-        "table",
-        Optional.empty());
+    Query query = new Query();
```

Comment:
Here, we could use the PostgresQuery instead of the Query, hence enabling us to break the dependency with the baremaps-config module. That was my intent when introducing this domain object.

Reply:
`baremaps-config` is no more ;)

I would rather use the generated model and move the openapi interface implementation in another module to prevent dependency conflicts.

Reply:
> `baremaps-config` is no more ;)

Fantastic 👏  🥳 

> I would rather use the generated model and move the openapi interface implementation in another module to prevent dependency conflicts.

I don't want to couple the baremaps-tile-postgres module with the baremaps-openapi module. In my understanding, TileJSON is not the only format and the PostgresQuery class will give us more flexibility.